### PR TITLE
Remove memory mx bean

### PR DESCRIPTION
**Pull request by oflebbe**
While looking into porting to iOS I discovered that the problematic method `getUsedMBAfterGC` isn't used anywhere. Proposing to remove it.

### CODE REVIEW

**Comment by easbar** [comment]
Ok its only used in downstream dependencies, but its easy to copy and paste wherever its needed and if its making problems here sure lets remove it.

File: pom.xml [hunk 1]
```diff
@@ -184,7 +184,7 @@
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>2.22.2</version>
                     <configuration>
-                        <argLine>-Xmx130m -Xms130m</argLine>
+                        <argLine>-Xmx130m -Xms130m -Duser.language=en</argLine>
```
Comment (easbar) [inline pom.xml:187]: Sorry I did not follow the language discussion fully, is this related here?

Comment (oflebbe) [reply pom.xml:187]: Sorry, forgot to clean history here before submitting. Will rebase the patch.

Comment (easbar) [reply pom.xml:187]: Ok but if you are rebasing this we can just as well just merge #2063 :) ?

Comment (oflebbe) [reply pom.xml]: Right, since it is included

**Comment by oflebbe** [comment]
Please consider  #2063 instead
